surefire-providers/common-junit4/src/main/java/org/apache/maven/surefire/common/junit4/JUnit4RunListener.java

```diff
@@ -128,11 +128,11 @@ protected StackTraceWriter createStackTraceWriter( Failure failure )
     @SuppressWarnings( { "UnusedDeclaration" } )
     public void testAssumptionFailure( Failure failure )
     {
-        this.reporter.testAssumptionFailure( createReportEntry( failure.getDescription() ) );
+        this.reporter.testAssumptionFailure( SimpleReportEntry.ignored( getClassName( failure.getDescription() ) , 
```

Comment:
@jimma 
Can you please introduce a new constructor `public SimpleReportEntry( String source, String name, String message )` and call it instead of `SimpleReportEntry.ignored`? Can you call that constructor in `SimpleReportEntry.ignored` as well? Thx.